rest/.jvm/src/main/scala/io/udash/rest/RestServlet.scala

```diff
@@ -47,27 +47,32 @@ class RestServlet(
 
   override def service(request: HttpServletRequest, response: HttpServletResponse): Unit = {
     val asyncContext = request.startAsync()
+    val completed = new AtomicBoolean(false)
+
+    // need to protect asyncContext from being completed twice because
```

Comment:
I'd mention that this happens due to timeout being set (or even add a link if you found sth relevant, e.g. https://stackoverflow.com/a/27744537).